File: module1/src/main/java/com/example/module1/MainActivity.java
package com.example.module1;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Context;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.WindowManager;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        WindowManager windowManager= (WindowManager) getSystemService(Context.WINDOW_SERVICE);



    }
}

File: appkotlin/src/main/java/com/example/appkotlin/activity/main/MainActivity.kt
package com.example.appkotlin.activity.main

import android.os.Bundle
import com.google.android.material.bottomnavigation.BottomNavigationView
import androidx.appcompat.app.AppCompatActivity
import androidx.navigation.findNavController
import androidx.navigation.ui.setupWithNavController
import com.example.appkotlin.R

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val navView: BottomNavigationView = findViewById(R.id.nav_view)

        val navController = findNavController(R.id.nav_host_fragment)
//        val appBarConfiguration = AppBarConfiguration(setOf(
//                R.id.navigation_home,
//                R.id.navigation_anti,
//                R.id.navigation_doctor,
//                R.id.navigation_follow,
//                R.id.navigation_setting))
//        setupActionBarWithNavController(navController, appBarConfiguration)
        navView.setupWithNavController(navController)


    }

//    suspend fun getBanner() {
//        RetrofitTest.getApi()?.getBanner()
//    }
}